.github/workflows/release.yml

```
@@ -0,0 +1,148 @@
+        run: npm run lint
+
+      - name: Type check
+        run: npm run typecheck
+
+      # Note: Add test step once tests are implemented
+      # - name: Run tests
+      #   run: npm test
+
+      - name: Get version from tag or input
+        id: get_version
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:3:30: Double quote to prevent globbing and word splitting

(shellcheck)